test/e2e/helpers/kafka_helper.go

```diff
@@ -44,25 +46,34 @@ var (
 )
 
 func MustPublishKafkaMessage(client *testlib.Client, bootstrapServer string, topic string, key string, headers map[string]string, value string) {
-	cgName := topic + "-" + key
+	cgName := topic + "-" + key + "z"
 
-	_, err := client.Kube.Kube.CoreV1().ConfigMaps(client.Namespace).Create(&corev1.ConfigMap{
+	cm := &corev1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      topic + "-" + key,
+			Name:      cgName,
 			Namespace: client.Namespace,
 		},
 		Data: map[string]string{
 			"payload": value,
 		},
-	})
+	}
+	_, err := client.Kube.Kube.CoreV1().ConfigMaps(client.Namespace).Create(cm)
 	if err != nil {
-		client.T.Fatalf("Failed to create configmap %q: %v", cgName, err)
-		return
+		if !apierrs.IsAlreadyExists(err) {
```

Comment:
That should never happen in theory, because the namespace is cleaned up on each test run...

Reply:
well keep it there for now

Reply:
I'm calling multiple times `MustPublishKafkaMessage` with the same `cgName`

Reply:
uh why?

Reply:
Because it fails when it receives multiple messages and randomly loses (2-4) / 5

Reply:
well, that's the behaviour we wanna find right? I'm not sure i fully get the problem